components/server/src/orgs/organization-service.spec.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { DBUser, TypeORM, UserDB, testContainer } from "@gitpod/gitpod-db/lib";
+import { DBTeam } from "@gitpod/gitpod-db/lib/typeorm/entity/db-team";
+import { User } from "@gitpod/ide-service-api/lib/ide.pb";
+import * as chai from "chai";
+import { Container, ContainerModule } from "inversify";
+import "mocha";
+import { suite, test } from "mocha-typescript";
+import { OrganizationService } from "./organization-service";
+import { v4 as uuidv4 } from "uuid";
+import { SpiceDBClient } from "../authorization/spicedb";
+import { SpiceDBAuthorizer } from "../authorization/spicedb-authorizer";
+import { Authorizer } from "../authorization/authorizer";
+import { v1 } from "@authzed/authzed-node";
+import { Experiments } from "@gitpod/gitpod-protocol/lib/experiments/configcat-server";
+
+const expect = chai.expect;
+
+@suite()
+export class OrganizationServiceSpec {
+    private container: Container;
+    private user: User;
+
+    async before() {
+        this.container = testContainer.createChild();
+        this.container.load(
+            new ContainerModule((bind) => {
+                bind(OrganizationService).toSelf().inSingletonScope();
+                bind(SpiceDBClient)
+                    .toDynamicValue(() => {
+                        const token = uuidv4();
+                        return v1.NewClient(token, "localhost:50051", v1.ClientSecurity.INSECURE_PLAINTEXT_CREDENTIALS)
+                            .promises;
+                    })
+                    .inSingletonScope();
```

Comment:
Will this result in a new client for each test? That's what we'd want, as we want to isolate each test from another test.

Reply:
yes, new client with unique token